.github/workflows/build-wheels.yml

```diff
@@ -32,24 +32,18 @@ jobs:
       - uses: actions/setup-python@v2
 
       - name: Install cibuildwheel
-        run: |
-          echo {{ runner.os }}
-          if [[ "{{ runner.os }}" == "Windows" ]]; then
-             sudo rm /usr/bin/link
-          fi
-          python -m pip install cibuildwheel==2.9.0
-        shell: bash
+        run: python -m pip install cibuildwheel==2.9.0
 
       - name: Build wheels
-        run: |
-          if [[ "{{ runner.os }}" == "Linux" ]]; then
-             for image in manylinux2010 manylinux2014; do
-                export CIBW_MANYLINUX_X86_64_IMAGE=$image
-                python -m cibuildwheel --output-dir wheelhouse           
-             done
-          else
-             python -m cibuildwheel --output-dir wheelhouse
-          fi
+        run: python -m cibuildwheel --output-dir wheelhouse
```

Comment:
Is this just a temporary change while debugging? I have found that manylinux2010 wheels are still needed on older HPC clusters, so I'd like to include them if possible

Reply:
This is a temporary change. These lines don't work on Windows, because running the build through bash does not work. Will need to figure out how to do this with a build matrix.

Reply:
Ok. I thought I'd got past that hurdle by setting `shell: bash` in the workflow step.

Reply:
Yes, but that enables bash on Windows - which has this problem: https://github.com/ilammy/msvc-dev-cmd#name-conflicts-with-shell-bash